api-gateway/src/main/kotlin/com/saveourtool/save/gateway/utils/StoringServerAuthenticationSuccessHandler.kt

```diff
@@ -24,6 +25,7 @@ class StoringServerAuthenticationSuccessHandler(
 ) : ServerAuthenticationSuccessHandler {
     private val logger = LoggerFactory.getLogger(javaClass)
     private val objectMapper = ObjectMapper()
+        .configure(SerializationFeature.WRITE_EMPTY_JSON_ARRAYS, false)
```

Comment:
Is it still needed? I think it causes empty arrays to be completely absent in JSONs, but how are non-empty arrays represented?

Reply:
At the time of user creation, this array will always be empty. Otherwise, we will receive the Kotlin.EmptyClass class in Json and will not be able to serialize it.

Reply:
Please add a comment in code about why we need this feature (btw its `EmptyList`, not `EmptyClass`) and add explicit initialization of `originalLogins` to `emptyList` in `fun Authentication.toUser()`. This way we probably won't accidentally change it to something different

Reply:
Ok. Fixed